server/handlers/handlers.go

```diff
@@ -255,11 +245,11 @@ func (ah *ActivityHandler) handleCreatedActivity(activityIds msteams.ActivityIds
 	newPost, appErr := ah.plugin.GetAPI().CreatePost(post)
 
 	if appErr != nil {
-		ah.plugin.GetAPI().LogError("Unable to create post", "post", post, "error", appErr)
+		ah.plugin.GetAPI().LogError("Unable to create post", "error", appErr)
```

Comment:
Just a question. Why are we removing these?

Reply:
these logs contain post details like messages, which we don't want to log